packages/icons/src/svg/overview.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <g transform="translate(0 1.388)">
+    <path d="M15.4028,12.6959 L0.5978,12.6959 L0.5978,3.2669 L8.5418,3.2669 L8.5418,3.2709 L11.7958,3.2709 L11.7958,3.2669 L15.4028,3.2669 L15.4028,12.6959 Z M1.5638,0.8149 C2.0118,0.8149 2.3758,1.1779 2.3758,1.6269 C2.3758,2.0759 2.0118,2.4399 1.5638,2.4399 C1.1148,2.4399 0.7508,2.0759 0.7508,1.6269 C0.7508,1.1779 1.1148,0.8149 1.5638,0.8149 L1.5638,0.8149 Z M4.2508,0.8149 C4.6988,0.8149 5.0628,1.1779 5.0628,1.6269 C5.0628,2.0759 4.6988,2.4399 4.2508,2.4399 C3.8018,2.4399 3.4378,2.0759 3.4378,1.6269 C3.4378,1.1779 3.8018,0.8149 4.2508,0.8149 L4.2508,0.8149 Z M6.9998,0.8149 C7.4488,0.8149 7.8128,1.1779 7.8128,1.6269 C7.8128,2.0759 7.4488,2.4399 6.9998,2.4399 C6.5508,2.4399 6.1878,2.0759 6.1878,1.6269 C6.1878,1.1779 6.5508,0.8149 6.9998,0.8149 L6.9998,0.8149 Z M14.8798,-0.0001 L1.1198,-0.0001 C0.5038,-0.0001 -0.0002,0.5049 -0.0002,1.1219 L-0.0002,13.2929 L15.9998,13.2929 L15.9998,1.1219 C15.9998,0.5049 15.4958,-0.0001 14.8798,-0.0001 L14.8798,-0.0001 Z"/>
+    <polygon points="1.907 11.074 6.892 11.074 6.892 4.852 1.907 4.852"/>
```

Comment:
those should be rect see https://developer.mozilla.org/en-US/docs/Web/SVG/Element/rect